### PR TITLE
Update Unit Test Coverage by 10%

### PR DESCRIPTION
**Pull request by drosenberg123**
Coverage was increased by 10% in each major category (Statements, Branches, Functions, Lines)

Please see [this bot post](https://github.com/CurriculumAssociates/createjs-accessibility/pull/93#issuecomment-1147793423) for delta info.

### CODE REVIEW

File: src/RoleObjects/ComboBoxData.test.js [hunk 1]
```diff
@@ -40,12 +40,14 @@ describe('ComboBoxData', () => {
...
         expect(ariaExpanded).toEqual(isExpanded);
       });
 
       it('sets "aria-readonly" attribute', () => {
-        const ariaReadOnly = comboBoxEl.getAttribute('aria-readonly') === 'true';
+        const ariaReadOnly =
+          comboBoxEl.getAttribute('aria-readonly') === 'true';
         expect(ariaReadOnly).toEqual(isReadOnly);
       });
     });
```
Comment (drosenberg123) [inline src/RoleObjects/ComboBoxData.test.js:53]: - prettier auto-fixes

File: src/RoleObjects/MenuItemData.test.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ describe('MenuItemData', () => {
   describe('register role', () => {
     let cjsMenuItem;
     let cjsCell;
-    let ilEl;
+    let liEl;
```
Comment (drosenberg123) [inline src/RoleObjects/MenuItemData.test.js:9]: - (trivial) typo

File: src/RoleObjects/ScrollBarData.js [hunk 3]
```diff
@@ -79,8 +79,8 @@ export default class ScrollBarData extends RangeData {
         this._displayObject.dispatchEvent(event);
       }
     } else if (
-      evt.keyCode === KeyCodes.left
-      || evt.keyCode === KeyCodes.right
+      evt.keyCode === KeyCodes.left ||
+      evt.keyCode === KeyCodes.right
```
Comment (drosenberg123) [inline src/RoleObjects/ScrollBarData.js:83]: - prettier auto-fix

File: src/Roles.js [hunk 4]
```diff
@@ -230,11 +230,11 @@ function getTagNameForDisplayObject(displayObject) {
...
-    || tagName === 'input'
-    || tagName === 'textarea'
-    || tagName === 'button'
+    tagName === 'a' ||
+    tagName === 'select' ||
+    tagName === 'input' ||
+    tagName === 'textarea' ||
+    tagName === 'button'
   );
 }
```
Comment (drosenberg123) [inline src/Roles.js:240]: - prettier auto-fixes

File: src/__jestSharedSetup.js [hunk 5]
```diff
@@ -33,6 +33,4 @@ afterEach(() => {
   }
 });
 
-export {
-  canvasEl, container, parentEl, stage,
-};
+export { canvasEl, container, parentEl, stage };
```
Comment (drosenberg123) [inline src/__jestSharedSetup.js:36]: - prettier auto-fix

File: src/index.js [hunk 6]
```diff
@@ -131,7 +132,9 @@ function register(configObjects) {
     objects = [configObjects];
   }
 
-  const accessiblityObjects = objects.map(objectConfig => createAccessibilityObjectForRole(objectConfig));
+  const accessiblityObjects = objects.map((objectConfig) =>
+    createAccessibilityObjectForRole(objectConfig)
+  );
 
   return accessiblityObjects.length > 1
     ? accessiblityObjects
```
Comment (drosenberg123) [inline src/index.js:140]: - prettier auto-fixes

File: src/RoleObjects/OrderedListData.test.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+          stage.accessibilityTranslator.update();
+          cjsOrderedList.accessible.addChildAt(cjsCell, 0);
+        }).toThrowError(errorObj);
+      });
+
+      const cjsDummy = new createjs.Shape();
+      AccessibilityModule.register({
+        displayObject: cjsDummy,
+        role: AccessibilityModule.ROLES.LISTITEM,
+      });
```
Comment (mmoss) [inline src/RoleObjects/OrderedListData.test.js:67]: What're these lines doing just hanging out? If they're needed for the next two cases you can just wrap the two cases in a describe, and throw them in a new `before`?

File: src/RoleObjects/ProgressBarData.test.js [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+        );
+      });
+
+      it('sets "value" attribute', () => {
+        expect(parseInt(progressEl.getAttribute('value'), 10)).toEqual(
+          valueNowVal
+        );
+      });
+
+      //  below props are inherited from RangeData
```
Comment (mmoss) [inline src/RoleObjects/ProgressBarData.test.js:60]: I'd recommend making this a describe:
e.g.
```
describe('inherits props from RangeData', () => {
  ...
});

File: src/RoleObjects/RowData.test.js [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+      it('sets "aria-rowindex" attribute', () => {
+        expect(parseInt(trEl.getAttribute('aria-rowindex'), 10)).toEqual(
+          rowIndexVal
+        );
+      });
+    });
+
+    describe('children checking', () => {
+      const errorObj =
```
Comment (mmoss) [inline src/RoleObjects/RowData.test.js:80]: Nit: Move into a `beforeEach` for this describe.

File: src/RoleObjects/RowData.test.js [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+      });
+
+      it('throws error attempting to add prohibited child using addChildAt()', () => {
+        expect(() => {
+          stage.accessibilityTranslator.update();
+          cjsRow.accessible.addChildAt(cjsSpan, 0);
+        }).toThrowError(errorObj);
+      });
+
+      const cjsDummy = new createjs.Shape();
```
Comment (mmoss) [inline src/RoleObjects/RowData.test.js:96]: Same as above, I'd move this into a `before` or `beforeEach` for the describe.

File: src/RoleObjects/SingleSelectListBoxData.test.js [hunk 11]
```diff
@@ -0,0 +1,235 @@
...
+        cjsListBox.accessible.required = newVal;
+        expect(cjsListBox.accessible.required).toEqual(newVal);
+      });
+
+      describe('can read and set "selected" property', () => {
+        const cjsListItem = new createjs.Shape();
+        AccessibilityModule.register({
+          displayObject: cjsListItem,
+          role: AccessibilityModule.ROLES.SINGLESELECTOPTION,
+        });
```
Comment (mmoss) [inline src/RoleObjects/SingleSelectListBoxData.test.js:191]: Let's pop these into a `before` as well 🙂

File: src/RoleObjects/SingleSelectListBoxData.test.js [hunk 12]
```diff
@@ -0,0 +1,235 @@
...
+          stage.accessibilityTranslator.update();
+          cjsListBox.accessible.addChildAt(cjsSpan, 0);
+        }).toThrowError(errorObj);
+      });
+
+      const cjsDummy = new createjs.Shape();
+      AccessibilityModule.register({
+        displayObject: cjsDummy,
+        role: AccessibilityModule.ROLES.SINGLESELECTOPTION,
+      });
```
Comment (mmoss) [inline src/RoleObjects/SingleSelectListBoxData.test.js:87]: Same as above, let's move these into a `before` so there's never any question when the lines execute.

File: src/RoleObjects/SingleSelectListBoxData.test.js [hunk 13]
```diff
@@ -0,0 +1,235 @@
...
+
+      it('sets "name" attribute', () => {
+        expect(ulEl.getAttribute('name')).toEqual(nameVal);
+      });
+    });
+
+    describe('children checking', () => {
+      const errorObj =
+        /Children of singleselectlistbox must have a role of singleselectoption/;
```
Comment (mmoss) [inline src/RoleObjects/SingleSelectListBoxData.test.js:68]: Nit: Pull into a `before`.

**Review by mmoss — DISMISSED** [review]

**Review by athottungal — APPROVED** [review]
LGTM!

**Pull request merged**